docs/UserGuide.md

```diff
@@ -40,7 +48,7 @@ HealthSync is a **powerful desktop application designed specifically for clinic
 
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
 
-   ![Ui](images/quickstartGUI.png)
```

Comment:
Is `images/quickstartGUI.png` still used?

Reply:
Yeah, there are still Screenshots for v1.3GUI in the repo. Do i remove all of them?

Reply:
Yes, please do!